src/utils/number.util.ts

```diff
@@ -6,6 +6,9 @@ export const formatAssetAmount = (
   decimalPlace = 6
 ) => amount.decimalPlaces(amount.abs().lt(1000) ? decimalPlace : 2, roundingMode).toFixed();
 
+export const roundFiat = (amount: BigNumber, roundingMode: BigNumber.RoundingMode = BigNumber.ROUND_DOWN) =>
```

Comment:
rename to `formatFiatAmount` please
to make it compatible with `formatAssetAmount`

Reply:
I'm afraid it cannot be named `format...` because a BigNumber instead of string is returned